client/src/App.tsx

```diff
@@ -1,22 +1,26 @@
 import React from 'react';
 import { BrowserRouter as Router, Route } from 'react-router-dom';
-import { Container } from '@material-ui/core';
-import { Home } from './pages/Home';
-import { Login } from './pages/Login';
+// import { Container } from '@material-ui/core';
```

Comment:
In my opinion, this container is needed, because with it our application can be scaled well on other devices.